creator-node/scripts/start.sh

```diff
@@ -80,6 +80,9 @@ if [[ "$devMode" == "true" ]]; then
         npx nodemon --exec 'node --inspect=0.0.0.0:${debuggerPort} --require ts-node/register src/index.ts' --watch src/ | tee >(logger)
     fi
 else
+    # Remove tmp track artifacts before starting the server so there are no incoming uploads
+    rm -rf /file_storage/files/tmp_track_artifacts
```

Comment:
@dmanjunath is this _always_ going to be the tmp track artifacts folder? In code I can see that `storagePath` is a config

Reply:
as far as i know yes. technically it should be `getConfigStoragePath()/files/tmp_track_artifacts` where getConfigStoragePath is from https://github.com/AudiusProject/audius-protocol/blob/4ca1c5414f956931f8d1afab99a4c56e5e9dcd65/creator-node/src/diskManager.js#L31